### PR TITLE
Support Django5+

### LINKED ISSUE #606: Support django 5+
@iMerica do you plan in the future to update support for Django 5+ ?

### PR DESCRIPTION
**Pull request by q0w**
Closes #606

### CODE REVIEW

File: dj_rest_auth/tests/requirements.pip [hunk 1]
```diff
@@ -1,7 +1,7 @@
 coveralls==1.11.1
 django-allauth==0.61.1
-django>=2.2,<5.0
-djangorestframework-simplejwt==4.6.0
+django>=2.2
+djangorestframework-simplejwt>=5.3.1
```
Comment (q0w) [inline dj_rest_auth/tests/requirements.pip:4]: https://github.com/jazzband/djangorestframework-simplejwt/issues/756
without this
```shell
ImportError: Could not import 'dj_rest_auth.jwt_auth.JWTCookieAuthentication' for API setting 'DEFAULT_AUTHENTICATION_CLASSES'. ImportError: cannot import name 'utc' from 'django.utils.timezone'
```

File: dj_rest_auth/tests/test_serializers.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 from allauth.socialaccount.providers.facebook.views import FacebookOAuth2Adapter
 from allauth.socialaccount.providers.facebook.views import FacebookProvider
 from allauth.socialaccount.models import SocialApp
-from allauth.exceptions import ImmediateHttpResponse
+from allauth.core.exceptions import ImmediateHttpResponse
```
Comment (q0w) [inline dj_rest_auth/tests/test_serializers.py:5]:
```shell
allauth.exceptions is deprecated, use allauth.core.exceptions
```

**Comment by q0w** [comment]
Current CI uses only python3.8 for testing. Maybe migrate to GHA all the way?

**Comment by awwester** [comment]
@q0w is it necessary to update simplejwt when allowing Django 5? #607 just updates the django requirement and passes CI tests.

**Comment by q0w** [comment]
@awwester yes, it's necessary

https://github.com/iMerica/dj-rest-auth/pull/610#discussion_r1551651958

**Comment by vasilistotskas** [comment]
Any updates on this ?

**Comment by iMerica** [comment]
Waiting on @q0w to address all feedback.

**Comment by q0w** [comment]
I already did
The current CI setup does not allow installing Django 5.
I think circuit should be rewritten to support all python versions or dropped in favour of GHA

**Comment by iMerica** [comment]
Good call, I'm going to switch this repo to use GHA.

**Pull request closed without merging**